### PR TITLE
fix: Use AsyncSession in memory

### PR DESCRIPTION
**Pull request by cbornet**
Use AsyncSession in memory

### CODE REVIEW

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
Hey @cbornet 

Have you checked the `ComponentTestBase`(https://github.com/langflow-ai/langflow/blob/a7aa3ab03f309e8303dc230b1816d14f6d322e2f/src/backend/tests/unit/components/inputs/test_input_components.py#L10)? 

Maybe we can build these for the components you've changed to make sure they don't break in future releases.

**Comment by cbornet** [comment]
@ogabrielluiz I fixed `TestChatInput`. Can you check again ?

**Comment by ogabrielluiz** [comment]
Hey @cbornet 

This is a very dangerous PR so we'll wait until next week to merge it. Just in case we need to release a patch this week.

**Review by ogabrielluiz — APPROVED** [review]

**Pull request merged**
